.github/test-config.yaml

```diff
@@ -10,10 +10,10 @@ excluded-charts:
   - charts/datarepo-api
   - charts/datarepo-ui
   - charts/oidc-proxy
+  - charts/gcloud-sqlproxy
+  - charts/create-secret-manager-secret
+  - charts/de-elasticsearch
```

Comment:
Per @tcjiang9 , since we already test the full datarepo umbrella chart, don't test individual charts that are part of the umbrella chart (of which `elasticsearch` is a part of). Otherwise we end up testing these charts multiple times.

Reply:
I'm actually wrong about this, for some reason, some charts such as create-secret-manager-secret only get tested when they're standalone and not when it's part of the umbrella, whereas datarepo-api DOES get tested as part of the umbrella. I have no idea why the umbrella only tests some of its dependencies and not others. I made a note to myself to investigate this further.